src/map/mob.cpp

```diff
@@ -1695,9 +1736,16 @@ int32 mob_randomwalk(struct mob_data *md,t_tick tick)
  */
 int32 mob_warpchase(struct mob_data *md, struct block_list *target)
 {
-	struct npc_data *warp = nullptr;
+	block_list *warp = nullptr;
 	int32 distance = AREA_SIZE;
-	if (!(target && battle_config.mob_ai&0x40 && battle_config.mob_warp&1))
+
+	int32 type = BL_NUL;
+	if (battle_config.mob_warp&1)
+		type |= BL_NPC;
+	if (battle_config.mob_warp&2)
+		type |= BL_SKILL;
+
+	if (!(target && battle_config.mob_ai&0x40 && type != BL_NUL))
```

Comment:
0x40 needs an update in conf/battle/monster.conf it only talks about NPC warps.

Reply:
Done!